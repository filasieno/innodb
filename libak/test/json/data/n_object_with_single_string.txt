name=n_object_with_single_string
max_depth=32
max_string_size=2048
max_json_size=1048576
----------
{ "foo" : "bar", "a" }