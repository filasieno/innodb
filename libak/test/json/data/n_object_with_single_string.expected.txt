---
STATE_CHANGED_EVENT: STATE_INITIALIZED
BEGIN_OBJECT
ATTR_KEY "foo" more=0
STRING_VALUE "bar" more=0
ATTR_KEY "a" more=0
STATE_CHANGED_EVENT: STATE_ERROR 224
